docs/index.md

```diff
@@ -505,6 +505,7 @@ Available Commands:
   diff         Diff releases defined in state file
   fetch        Fetch charts from state file
   help         Help about any command
+  init         Checks and installs deps
```

Comment:
deps? There is ambiguity.

Reply:
Change to: Initialize the helmfile, check the helm version and plugins